scripts/discover/as-host-discovery

```diff
@@ -0,0 +1,50 @@
+#! /bin/bash
```

Comment:
`as-discover-ips` and `as-host-discovery` seem VERY similar. Do we need both?

Reply:
they are VERY similar, but i wanted a away to test new things without messing up how its working currently. I could see a in the future having as-discover-domains, and switching as-host-discovery to call as-discover-ips and as-discover-domains.